modules/standard/Math.chpl

```diff
@@ -46,10 +46,10 @@ exception will be generated.
 Constants
 ---------
 :var:`e`
-:var:`log2_e`
-:var:`log10_e`
-:var:`ln_2`
-:var:`ln_10`
+:var:`log2E`
```

Comment:
These use ':var:' while the actual definitions use ':param:'. I am guessing the documentation works regardless, but it would be nice to have both match

Reply:
It does, but I'll update them to be more correct anyways.  Thanks!